install/third-party/dbmarlin/install.yml

```diff
@@ -0,0 +1,13 @@
+id: third-party-dbmarlin
+name: dbmarlin
+title: DBmarlin
+description: |
+  View database performance from within New Relic with this integration with DBmarlin.
+target:
+  type: integration
+  destination: cloud
+
+install:
+  mode: link
+  destination:
+    url: https://www.dbmarlin.com/partners
```

Comment:
This is the link that user's will be sent to during installation, we might want to change it over to a link to documentation on how to setup the integration. The page at `https://www.dbmarlin.com/partners` makes no reference is such an integration.